src/brouwers/kitreviews/models.py

```diff
@@ -21,28 +24,28 @@ class KitReview(models.Model):
     raw_text = models.TextField(
         _(u'review'),
         help_text=_('This is your review. You can use BBCode here.')
-        )
+    )
     html = models.TextField(blank=True, help_text=u'raw_text with BBCode rendered as html')
-    positive_points = models.TextField(_('positive points'), blank=True)
-    negative_points = models.TextField(_('negative points'), blank=True)
-    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING)
+    prop = models.ManyToManyField('KitReviewProperty', blank=True, related_name='reviews')
```

Comment:
can you rename this to `properties`? Always pluralize m2m and only shorten variable/field names if it's really needed.

For the `related_name` you can set it to `'+'` which effectively disables it (see https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.ForeignKey.related_name). It does not make much sense to look up all the attached kitreviews for a property, as it *should* be all of them.